animation-frames-timing-function/index-waa.html

```diff
@@ -23,37 +23,39 @@
         left: 10px;
         color: white;
       }
-
     </style>
   </head>
   <body>
-    <div class="div1"><p>transition-timing-function: frames(10)<p></div>
-    <div class="div2"><p>transition-timing-function: steps(10);<p></div>
-    <div class="div3"><p>transition-timing-function: ease-in;<p></div>
+    <div class="div1">
+      <p>transition-timing-function: frames(10)</p>
```

Comment:
`frames(10)` => `steps(5)`?

Reply:
Yes, Either is ok. I started out recently so I don't know that there was a ```frames()``` function ,as i couldn't find it anywhere. I forgot to change in the text in html.

Reply:
```suggestion
      <p>transition-timing-function: steps(5)</p>
```